src/libs/actions/PersonalDetails.ts

```diff
@@ -99,8 +100,8 @@ function extractFirstAndLastNameFromAvailableDetails({login, displayName, firstN
         return {firstName: '', lastName: ''};
     }
 
-    const firstSpaceIndex = displayName.indexOf(' ');
-    const lastSpaceIndex = displayName.lastIndexOf(' ');
+    const firstSpaceIndex = _.indexOf(displayName, ' ');
+    const lastSpaceIndex = _.lastIndexOf(displayName, ' ');
```

Comment:
I think we no longer use lodash in ts files

Reply:
cool, optional chaining it is then, as `displayName` can be `undefined`